model/Vocabulary.php

```diff
@@ -632,4 +632,20 @@ public function getId() {
       return $this->config->getId();
     }
 
+    public function getModifiedDate()
+    {
+        $modified = null;
+        // finding the modified properties
+        /** @var \EasyRdf\Resource $modifiedResource */
+        $modifiedResource = $this->resource->get('dc:modified');
+        if ($modifiedResource) {
```

Comment:
For a Vocabulary, I am simply getting its `dc:modified`. Should we use the concept main scheme for a Vocabulary too @osma ? Or, should we fallback to the Vocabulary's `dc:modified` in the `Concept#getModifiedDate`, if no `dc:modified` in the Concept, and no `dc:modified` in the Concept main scheme?